src/coda/apps/templates/invoices/detail.html

```diff
@@ -11,25 +11,102 @@ <h1 class="my-2">
                 <small class="pill status-label rejected">{{ invoice.status }}</small>
             {% endif %}
         </h1>
-        <a href="{% url "invoices:update" pk=invoice.id %}"
-           role="button"
-           class="secondary">Edit</a>
+        <div>
+            <button hx-post="{% url 'invoices:pay_invoice' pk=invoice.id %}"
+                    hx-trigger="click"
+                    hx-swap="outerHTML"
+                    hx-target=".pill"
+                    hx-on="htmx:afterRequest: window.location.reload()"
```

Comment:
If we're doing a client side reload here then we may as well do it on the server instead